homography_matrix:
  cols: 3
  data:
  - - 1.348690564359867
    - 0.5620146048383169
    - -426.89217072551816
  - - -0.026887926081221553
    - 1.8358771735724853
    - -166.48175005629682
  - - 1.6582290252940743e-05
    - 0.0006032384343774633
    - 0.9999999999999999
  rows: 3
